src/scss/mofo-bootstrap.scss

```diff
@@ -9,18 +10,17 @@
 @import 'custom/colors';
 
 // Core variables and mixins
+@import '../../node_modules/bootstrap/scss/functions';
 @import '../../node_modules/bootstrap/scss/variables';
 @import './overrides/variables';
 @import '../../node_modules/bootstrap/scss/mixins';
-@import './overrides/mixins';
 
 // Reset and dependencies
-@import '../../node_modules/bootstrap/scss/normalize';
+@import '../../node_modules/bootstrap/scss/root';
```

Comment:
Where'd this come from, what does it do? Did bootstrap add it recently or was it something we intentionally left out previously?

Reply:
Normalize was dropped into reboot, here https://github.com/twbs/bootstrap/commit/4fa7749442ea7ab609d1c8c25dc4ba750194e542#diff-716327f6d9890462da5d6f981f62fcc6

Reply:
Seems like reboot/root are two different things?